options.html

```diff
@@ -7,16 +7,25 @@
     <style type="text/css">
       body {
           font: message-box;
-          text-rendering: optimizeLegibility;
           display: flex;
           flex-direction: column;
           max-width: 650px;
           margin: 0 auto;
       }
+      table {
+          margin-bottom: 3px;
+          width: 100%;
+      }
+      td:nth-child(1) {
+          width: 16em;
+      }
```

Comment:
```suggestion
```
No fixed width, please.

Reply:
Hmmm, it's not really fixed width... it's width is based on the size of the font being used.  I used this method because I didn't want to change more of your code than was required.  To this end, I made 1 column a reasonable width for the expected content, and allowed the other columns to automatically change their widths based on the size of the viewport (with the exception of the `select` element, which I made the minimum width to contain its known static content).

Reply:
True, but it's still giving the field a size that's not based on available space. Just apply the suggestion and remove it.

Reply:
I'll remove it, but it may cause empty space.